libsolidity/interface/Natspec.cpp

```diff
@@ -225,7 +225,7 @@ Json::Value Natspec::extractCustomDoc(multimap<string, DocTag> const& _tags)
 	for (auto const& [tag, value]: _tags)
 		if (boost::starts_with(tag, "custom"))
 			concatenated[tag] += value.content;
-	Json::Value result;
```

Comment:
This used to return `null` for the case that `_tags` is empty, right? Does this change anything in the output ("remove null values")?

Reply:
Yes, I was wondering about this myself, whether I should address it or not. We can skip this or add an explicit condition for aborting early.

Reply:
Does it change anything down the line?

Reply:
I don't think so. Looking at this file, everything that that uses this function either uses `.empty()` to skip it the value (which works the same for `null` and for arrays/objects) or just adds more keys ensuring it's not empty.

Reply:
This is used by two places: `devDocumentation(ContractDefinition)` and `devDocumentation(map<string, DocTag>)`. The second used by the first. In cases there was no fields to be added, it does check for not inserting empty values. So I guess we can keep this one as it was, and look at improving this entire NatSpec generator at some later time.